src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -54,7 +54,8 @@ void AbstractTableGenerator::generate_and_store() {
       auto& table = table_info_by_name[table_name].table;
       auto table_wrapper = std::make_shared<TableWrapper>(table);
       table_wrapper->execute();
-      auto sort = std::make_shared<Sort>(table_wrapper, table->column_id_by_name(column_name));
+      auto sort = std::make_shared<Sort>(table_wrapper, table->column_id_by_name(column_name), OrderByMode::Ascending,
+                                         _benchmark_config->chunk_size);
```

Comment:
The chunk size is not required, isn't it? Any reason to set it to this specific value (chunk_size * 10)?

Reply:
Where do you get `chunk_size * 10` from?

Reply:
Never mind, I confused our default chunk size with that of 10k here. Nonetheless, why setting it?

Reply:
Because Sort materializes the table and does not check for the size of the input table.